content/adr/angular-fe.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Adopt Angular for Frontend Development'
+type: "adrs"
```

Comment:
What does `type` do?

Reply:
The 'type' here would help us to get all ADRs and if we haven't created any ADRs then we could create a fallback condition like below - 
{{ $adrPages := where $.Site.RegularPages "Type" "adrs" }}

Reply:
Hmm... Is this a result of the earlier code where ADRs were of `docs` archetype? I am wondering why this is needed because we should be able to just look up all the content of archetype `adr`.

Reply:
@hussainweb I will go ahead remove the 'type' then for now and find a better way to list out all ADRs.